worker/src/main/scala/com/lucidchart/piezo/TriggerMonitoringModel.scala

```diff
@@ -27,7 +26,8 @@ class TriggerMonitoringModel(props: Properties) {
   val connectionProvider = new ConnectionProvider(props)
 
   def setTriggerMonitoringRecord(
-    trigger: Trigger,
+    triggerName: String,
```

Comment:
My above comment about passing in the name and group together also applies here.